source/OutfitterPanel.cpp

```diff
@@ -74,6 +87,19 @@ namespace {
 		}
 		return toRefill;
 	}
+
+
+
+	static bool IsLicense(const string &name)
+	{
+		static const string &LICENSE = " License";
```

Comment:
```suggestion
		static const string LICENSE = " License";
```
You allocate it anyway, right?